src/main/java/org/openrewrite/java/testing/assertj/JUnitAssertThrowsToAssertExceptionType.java

```diff
@@ -48,31 +49,33 @@ public String getDescription() {
     @Override
     public TreeVisitor<?, ExecutionContext> getVisitor() {
         return Preconditions.check(new UsesMethod<>(ASSERT_THROWS_MATCHER), new JavaIsoVisitor<ExecutionContext>() {
+
             @Override
             public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, ExecutionContext ctx) {
                 J.MethodInvocation mi = super.visitMethodInvocation(method, ctx);
                 if (!ASSERT_THROWS_MATCHER.matches(mi)) {
                     return mi;
                 }
 
-                if (!(getCursor().getParentTreeCursor().getValue() instanceof J.Block)) {
-                    return mi;
-                }
+                boolean returnActual = !(getCursor().getParentTreeCursor().getValue() instanceof J.Block);
 
                 maybeAddImport(ASSERTIONS_FOR_CLASS_TYPES, "assertThatExceptionOfType");
                 maybeRemoveImport(JUNIT_ASSERTIONS + ".assertThrows");
                 maybeRemoveImport(JUNIT_ASSERTIONS);
 
                 List<Expression> args = mi.getArguments();
+
+                UnaryOperator<String> decorator = returnActual ? code -> code + ".actual()" : UnaryOperator.identity();
```

Comment:
Would it not be easier to use the following here?
```java
if (returnActual) {
  code += ".actual()";
}

Reply:
Probably yeah 😅